maestro-orchestra/src/main/java/maestro/orchestra/workspace/WorkspaceExecutionPlanner.kt

```diff
@@ -17,6 +18,7 @@ object WorkspaceExecutionPlanner {
         input: Path,
         includeTags: List<String>,
         excludeTags: List<String>,
+        config: File?,
```

Comment:
To be consistent, I think `config` should be `Path` (just like `input` parameter above), instead of `File`.